src/main/java-templates/org/postgresql/ds/PGConnectionPoolDataSource.java

```diff
@@ -29,7 +29,7 @@
  * @author Aaron Mulder (ammulder@chariotsolutions.com)
  */
 public class PGConnectionPoolDataSource
-    extends @CONN_POOL_DS_CLASS@
+    extends ${CONN_POOL_DS_CLASS}
```

Comment:
JDBC4 is java 6, isn't it? I think we can safely make that a minimum. I think even java 7 as a minimum is good enough.